content/tagging/assigning_tags.md

```diff
@@ -21,7 +21,15 @@ There are several places tags can be assigned: [configuration files](#configurat
 
 ## Configuration Files
 
-Configure the host tags submitted by the Agent inside `datadog.yaml`. The tags for the [integrations][5] installed with the Agent are configured via YAML files located in the **conf.d** directory of the Agent install. To locate the configuration files, refer to [the Agent configuration files FAQ][6].
+The host tag is [assigned automatically][6] by the Datadog Agent. To customize the host tag, use the Agent configuration file, `datadog.yaml`.
```

Comment:
Shouldn't we refer to those as `host tags` since multiple host tags are allowed? I think we should also give a snippet of host tags configuration in the `datadog.yaml` file i.e: 

```
# Set the host's tags (optional)
tags:
  - <KEY>:<VALUE>   
  - env:prod  
  - role:database
```

Reply:
I think we should also precise that host tags are attached to all metrics, traces, and logs forwarded by a given Datadog Agent allowing for a better correlation between those data types within Datadog.

Reply:
This part is specific to the hostname (`host` tag). I updated to make that more clear.